### ISSUE TITLE
Training with 5 last digits

### ISSUE THREAD

**Issue by diegoalejogm** [hangyao/street_view_house_numbers#1]
Hi, I was looking at the training models in the file _4_CNN_multi.ipynb_ and the accuracy functions are compared against the last 5 digits of each image, instead of the first 5 digits, which are the ones that contain the most information. Could you explain why is it done in this way? I believe is an error.

**Comment by hangyao** [comment]
Hi @diegoalejogm ,

I saved the number of total digits of house numbers in tf_train_labels[:, 0] (according to the Goodfellow's paper), but eventually didn't use it for training the model. tf_train_labels[:, 1]  to tf_train_labels[:, 5] are 5 digits of house numbers. If you investigate the original data you will find that only one house number has 6 digits, and I removed that instance from the training data. Therefore all house numbers have 5 or less digits. For those house numbers with less than 5 digits, I filled the 'blank' digits with a class 10. I hope this can answer your question.
